packages/eslint-plugin/src/rules/no-misused-spread.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  ESLintUtils,
+  type TSESLint,
+  type TSESTree,
+} from '@typescript-eslint/utils';
+
+import { createRule } from '../util';
+
+export default createRule({
+  defaultOptions: [],
+  name: 'no-misused-spread',
+  meta: {
+    docs: {
+      description: 'Disallow spread operator on function',
+      recommended: 'stylistic',
+      requiresTypeChecking: true,
+    },
+    messages: {
+      forbidden:
+        'Spreading a function is almost always a mistake. Did you forgot to call the function?',
+    },
+    schema: [],
+    type: 'suggestion',
```

Comment:
IMO this is most of the time a problem, but untill today there is nothing warning the user that it is one.
So I guess this is an expected usage and I choose `suggestion` rather than `problem`.
I can change it if needed.

Reply:
Not sure... 
let's wait for @JoshuaKGoldberg I guess?

Reply:
It's a `'problem'`. Anything that catches directly incorrect _logic_ is a problem.